firefox-ios/Storage/Rust/RustFirefoxSuggestion.swift

```diff
@@ -21,6 +21,7 @@ public struct RustFirefoxSuggestion {
     public let iconImage: UIImage?
     public let fullKeyword: String
     public let clickInfo: RustFirefoxSuggestionInteractionInfo?
+    public let score: Float?
```

Comment:
Hmmm, was there a reason you added this to `RustFirefoxSuggestion`? It doesn't look like we're using the score anywhere else.

Reply:
oh good point